UKHomeOffice/cerberus-service
# COP-12143 Created the previous task list page

## Description
Created the previous tasks list page
![image](https://user-images.githubusercontent.com/12000340/199438635-3e6d42a0-5d6a-4e8d-9991-cc93d9340642.png)


## To Test
Go to this link with data filled in: http://localhost:8084/previous-tasks?movement-id={movementId}&type={type}&svxId=&pole-v1-id={poleV1Id}&pole-v2-id={poleV2Id}
for example http://localhost:8084/previous-tasks?movement-id=AUTOTEST-31-10-2022-RORO-Accompanied-Freight_784991&type=PERSON&svxId=&pole-v1-id=ROROXML:P=f0cd6a0235e1d4200c5335cf00ebeae2
Note: I think all data is stubbed right now so as long as you get the Ids correct you'll see the same data on every entity

## Developer Checklist

\* Required
